### PR TITLE
Am update assets

### PR DESCRIPTION
**Pull request by Oo1Insane1oO**
This suddenly got a bit big, but essentially adds functions for creating a set of assets with basevoltage added too the conducting ones and location x, y added to substation

Main point of this PR is to reflect the structure of the actual asset builder more. The old parts are still present in utils.

Also adds module for adding random (deterministic) parent ids to the assets. Currently only for GeographicalRegion <- BiddingArea <- Substation (i.e BiddingArea parent for Substation and GeographicalRegion parent for BiddingArea). Remaining hierarchy to come in future PR.

Also, the actual connectivity is not implemented yet, also for future PR.

### CODE REVIEW

**Comment by Oo1Insane1oO** [comment]
look at create_hierarchy, if you want too test the output dataframe, since this is the one I'll throw into the actual dummy model in the end

File: pyproject.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ packages = [
 ]
 
 [tool.black]
-line-length = 120
+line-length = 100
```
Comment (sanderland) [inline pyproject.toml]: 120 is our standard

File: Jenkinsfile [hunk 2]
```diff
@@ -59,6 +59,7 @@ podTemplate(
             }
             stage('Test') {
                 sh("poetry run pytest")
+                sh("poetry run pytest --doctest-modules powerdummy")
```
Comment (sanderland) [inline Jenkinsfile:62]: can we not have doctests?

Comment (Oo1Insane1oO) [reply Jenkinsfile:62]: i'll fix it in a future PR

File: pyproject.toml (hunk 1 above)
Comment (Oo1Insane1oO) [reply pyproject.toml]: will comply

**Review by sanderland — APPROVED** [review]

**Pull request merged**